lib/src/internal/http/http_route.dart

```diff
@@ -76,6 +76,9 @@ abstract class IHttpRoute {
   /// Adds the [`scheduled-events`](https://discord.com/developers/docs/resources/guild-scheduled-event#get-guild-scheduled-event) part to this [IHttpRoute].
   void scheduledEvents({String? id});
 
+  /// Adds the [`rule`](https://discord.com/developers/docs/resources/auto-moderation#auto-moderation) part to this [IHttpRoute].
```

Comment:
Should be rules (not rule) and should link to https://discord.com/developers/docs/resources/auto-moderation#get-auto-moderation-rule